code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -127,8 +128,9 @@
 		strength--
 		strength_change()
 
-		message_admins("PA Control Computer decreased to [strength] by [key_name_admin(usr)](<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) in ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
-		log_game("PA Control Computer decreased to [strength] by [key_name(usr)] in ([x],[y],[z])")
+		message_admins("PA Control Computer decreased to [strength] by \

```

Comment:
follow the goddamn guidelines; `Code where one line of code is split across mutiple lines (except for strings and comments and in those cases existing longer lines must not be split up)`